client/packages/system/src/Item/api/operations.graphql

```diff
@@ -60,6 +60,12 @@ fragment ItemRowWithStats on ItemNode {
   }
 }
 
+fragment ColdStorageType on ColdStorageTypeNode {
```

Comment:
This is in the item graphql file, but doesn't feel ideal. In someways it's nice though in that we do want this fragment here...

Reply:
Maybe location is the most ideal location, but I think here is fine too 🤷‍♀️